### PR TITLE
bugfix/small-fixes-nino

### PR DESCRIPTION
**Pull request by NinoRy**

### CODE REVIEW

File: modules/OverdueDebts/Resources/lang/de/messages.php [hunk 1]
```diff
@@ -7,6 +7,7 @@
     'csvImportActive' => 'Der OP-Import läuft gerade. Bitte warten Sie bis der Prozess beendet wurde.',
     'import' => [
         'block' => 'Blockiere Internetzugriff und füge Mahngebühren hinzu...',
+        'columnCountError' => 'Die Anzahl der Spalten in der CSV ist fehlerhaft. Die Anzahl muss 16 Spalten betragen.',
```
Comment (cschra) [inline modules/OverdueDebts/Resources/lang/de/messages.php]: number is static and should be changed to :number as well

File: app/Console/Commands/SyncLanguagesCommand.php [hunk 2]
```diff
@@ -47,7 +47,7 @@ public function handle()
 
         foreach ($languagefiles as $languagefile) {
             $originalPath = base_path('resources/lang/en/').$languagefile;
-            $originalArray = collect(require($originalPath));
+            $originalArray = collect(require $originalPath);
```
Comment (cschra) [inline app/Console/Commands/SyncLanguagesCommand.php]: These StyleCI issues are also fixed by @olebowle and already in dev I think - could you rebase to dev and check if they still appear

Comment (NinoRy) [reply app/Console/Commands/SyncLanguagesCommand.php]: done

File: modules/OverdueDebts/Resources/lang/de/messages.php (hunk 1 above)
Comment (NinoRy) [reply modules/OverdueDebts/Resources/lang/de/messages.php]: right ... done

**Pull request merged**